### PR TITLE
feat(LED-33): update LCO settings

### PR DESCRIPTION
**Pull request by perjo927**
Add support for customer.showShippingAddressContact

TODO: screenshot + manual test

### CODE REVIEW

**Review by ssolders — APPROVED** [review]
LGTM, but I think you forgot to update the changelog?

**Comment by perjo927** [comment]
> LGTM, but I think you forgot to update the changelog?

It's there :)

File: changelog.readme [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.3.0
```
Comment (ssolders) [inline changelog.readme:1]: 1.3.0 is activating refund, so bump to 1.4.0?
```suggestion
## 1.4.0
```

**Comment by ssolders** [comment]
> > LGTM, but I think you forgot to update the changelog?
> 
> It's there :)

Just me who's blind :)

File: changelog.readme (hunk 1 above)
Comment (perjo927) [reply changelog.readme:1]: Yes :)

**Pull request merged**